extensions/postgres/schema/xdb_sys_schema.sql

```diff
@@ -64,4 +64,12 @@ CREATE TABLE xdb_sys_timer_tasks(
     state_id_sequence INTEGER, -- for looking up xdb_sys_async_state_executions
     info jsonb ,
     PRIMARY KEY (shard_id, fire_time_unix_seconds, task_sequence)    
+);
+
+CREATE TABLE xdb_sys_local_queue(
+    process_execution_id uuid,
+    dedup_id uuid,
+    queue_name VARCHAR(31),
+    payload jsonb,
+    PRIMARY KEY (process_execution_id, dedup_id)
```

Comment:
@longquanzheng we actually don't need the queue_name in the PKs.

Reply:
Very good point! Agreed